### ISSUE TITLE
Colors & Contrasts

### ISSUE THREAD

**Issue by chenejac** [vivo-project/VIVO#4097]
**Is your feature request related to a problem? Please describe.**
VIVO should be aligned with AA level of WCAG regarding **colors and contrasts**.
